src/Language/Docker/PrettyPrint.hs

```diff
@@ -139,6 +139,12 @@ prettyPrintChown chown =
     Chown c -> "--chown=" <> pretty c
     NoChown -> mempty
 
+prettyPrintChmod :: Chmod -> Doc ann
```

Comment:
Is this being used anywhere?

Reply:
It is now. I completely forgot about the pretty printer yesterday after I had the parser working. That being said, the pretty printer is lacking tests in a big way.
I wonder if it even being used at all.

Reply:
I'm using it in some private projects, and one public. Thanks for fixing it.

I agree that this should have more tests.